### ISSUE TITLE
(offtopicish) should rdfa, microdata parser toolkits extract @alt?

### ISSUE THREAD

**Issue by danbri** [schemaorg/schemaorg#189]
Recording a spin-off question from https://github.com/rvguha/schemaorg/issues/158 - 

"We might also want to consider whether Microdata and RDFa parsers should be extracting the alt text as an extra/service, but that's another issue."

**Comment by danbri** [comment]
I was thinking out loud, @scor, but yes this could be propagated into the core specs or left as an added value non-standard flag within tools. The idea is to reduce the need to keep dropping back to the underlying HTML page to get things that weren't extracted as triples.

**Comment by danbri** [comment]
these ships have sailed, circumnavigated the globe, and returned home
